tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-distributed-test-ci-pipeline.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_Distributed_Test
+
+  timeoutInMinutes: 120
+  pool: 'Linux-Multi-GPU-V100'
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - template: templates/run-docker-build-steps.yml
+    parameters:
+      RunDockerBuildArgs: |
+        -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory) \
+        -x " \
+          --config RelWithDebInfo \
+          --enable_training \
+          --update --build \
+          --build_wheel \
+          " \
+        -m
```

Comment:
fyi, it may be useful to explicitly tag the image with run_dockerbuild.sh argument `-t <image_name>`. then, this name can be used to refer to the image later instead of run_dockerbuild.sh's default name, which could change.

Reply:
I'll add a tag to this image.